packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/BasicInformationStep/BasicInformationForm/BasicInformationForm.tsx

```diff
@@ -43,6 +43,7 @@ const BasicInformationForm = ({
   selectedCommunity,
   onSubmit,
   onCancel,
+  setSelectedChainId,
```

Comment:
good stuff, I changed in this particular case to `handleSelectedChainId` because I had other props named like this. as well as `on{action}`. Yeah we can try to decide when we want to use `on` and when `handle` and try to use this pattern. looks like `handle` could be a function that is used in the component and `on` when you want to pass it as a prop down to the child component.